contexts/wallet/index.js

```diff
@@ -124,26 +185,46 @@ export const useWallet = () => {
     }
   };
 
-  const signMessage = async (msg) => {
+  const signMessage = async (msg, accountId) => {
+    let provider;
+    if (!library) {
+      provider = await injected.getProvider();
+    } else {
+      provider = library.provider;
+    }
     try {
-      const signedMessage = await library.provider.request({
```

Comment:
here, I also use one of the `injected` methods, otherwise as with "Sign In with wallet" the library is not ready when we need it @juliencharrel